samples/unity-of-bugs/Assets/Scenes/1_BugfarmScene.unity

```diff
@@ -170,8 +170,8 @@ MonoBehaviour:
   m_Enabled: 1
   m_EditorHideFlags: 0
   m_Script: {fileID: 11500000, guid: 4e29b1a8efbd4b44bb3f3716e73f07ff, type: 3}
-  m_Name:
-  m_EditorClassIdentifier:
+  m_Name: 
```

Comment:
Not sure what caused all these white spaces. Tricky to revert the file while keeping the new buttons only.